web/services/AttachmentService.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const botName = require('../../package.json').name;
+const MAX_FILE_SIZE = 5000000;
+
+export default class AttachmentService {
+  constructor(bdk) {
+    this.bdk = bdk;
+  }
+
+  /**
+   * Create botAction for postAttachmentTimeline
+   *
+   * @param {string} f - file being uploaded
+   * @param {string} selectedChatter
+   */
+  async postAttachment(f, selectedChatter) {
```

Comment:
this isn't returning anything, if we're `await`ing this anywhere i think it will never resolve

Reply:
That actually makes sense, however isn't the test doing that & successfully resolving? https://github.com/salesforce/refocus-bot-timeline/pull/25/files#diff-6a85f2da74f8b9f91c71dcf999af20294a4f80b6fe0182ce5b0f214003a76bf3R27

I agree though, I'll put in the `return` in to make it more explicit

Reply:
Using `return await` is bad practice.  @kriscfoster  @ConalCosgrove 
The alternative is: 
`async function foo() {
    await bar();
    return;
}`

Ref:
https://eslint.org/docs/rules/no-return-await
http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea

Reply:
maybe add the rule to eslintrc?